libraries/AP_NavEKF3/AP_NavEKF3_PosVelFusion.cpp

```diff
@@ -759,7 +751,7 @@ void NavEKF3_core::FuseVelPosNED()
                 // If we are doing full aiding and velocity fusion times out, reset to the GPS velocity
                 if (PV_AidingMode == AID_ABSOLUTE && velTimeout) {
                     // reset the velocity to the GPS velocity
```

Comment:
Ah!  Interesting!  Look at this comment and the parameter you're passing through!

This is *assuming* that the default is GPS - which is something it ought not be doing.  So either the comment needs fixing or the call does.  

Same thing in the comment two lines up.

Just on its own this makes this PR a good one.

Reply:
In original code, velResetSource is DEFAULT when ResetVelocity() get called. If I pass resetDataSource::GPS, it will be different from current code

Reply:
In current code, velResetSource  could be anything (it can be set anywhere) when ResetVelocity() get called here.

Reply:
Yes.  So the easy fix is to change the comment.  If you really don't want to fiddle here I'm not going to hold up the PR for it.  At least it now stands out like a sore thumb.

Reply:
Thank you @peterbarker 
>  it now stands out like a sore thumb.

Yes, I agree.  But I found more and more comments above this line need to change,  if I change this comment.  Would you mind reconsidering parameter default? It will limit code change to minimum.  I just try to make posResetSource  and velResetSource local, because they are only used inside ResetVelocity() and ResetPosition(). It is for upcoming ext nav fix.

Reply:
Probably better to leave it as an NFC PR IMO.

Since you're still working in here I'm sure things will happen here anyway.

Reply:
Thank you @peterbarker 
I think you are right. I have fix the comments. Would you mind taking a look? Thank you

Reply:
@chobitsfan looks fine except "externel" should be "external".